src/Set/LaravelSetList.php

```diff
@@ -78,6 +78,11 @@ final class LaravelSetList implements SetListInterface
      */
     final public const LARAVEL_90 = __DIR__ . '/../../config/sets/laravel90.php';
 
+    /**
+     * @var string
+     */
+    final public const LARAVEL_10 = __DIR__ . '/../../config/sets/laravel10.php';
```

Comment:
Shouldn't it be `LARAVEL_100`?

Reply:
Good catch @alexey-m-ukolov 🙌 
I wouldn't have caught that.